src/main/java/it/gov/pagopa/pu/organization/repository/TaxonomyRepository.java

```diff
@@ -1,13 +1,22 @@
 package it.gov.pagopa.pu.organization.repository;
 
 import it.gov.pagopa.pu.organization.model.Taxonomy;
+import java.util.Optional;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 
-import java.util.Optional;
-
 @RepositoryRestResource(path = "taxonomies")
 public interface TaxonomyRepository extends JpaRepository<Taxonomy, Long> {
 
   Optional<Taxonomy> findByTaxonomyCode(String taxonomyCode);
+
+  @Query("SELECT distinct t from Taxonomy t WHERE " +
```

Comment:
why `distinct`? you are projecting the entire DTO... it cannot have duplicates...